src/Resources/contao/templates/backend/be_no_root.html5

```diff
@@ -16,8 +16,14 @@ $lang = (object) $GLOBALS['TL_LANG']['XPT'];
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
 
   <style><?php
-    include TL_ROOT . '/system/themes/' . $theme . '/basic.css';
-    include TL_ROOT . '/system/themes/' . $theme . '/error.css';
+    $includeIfExists = function ($file) {
```

Comment:
I don't agree. The Twig template loader might be a future feature but everything else is already there and could/should be used.